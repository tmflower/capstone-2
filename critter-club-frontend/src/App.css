/* CSS HEX */
/* --honeydew: #e8f7eeff;
--ash-gray: #b8c4bbff;
--dark-brown: #6c4d25ff;
--green-blue-crayola: #1e91d6ff;
--sap-green: #447604ff; */

/* CSS HSL */
/* --honeydew: hsla(144, 48%, 94%, 1);
--ash-gray: hsla(135, 9%, 75%, 1);
--dark-brown: hsla(34, 49%, 28%, 1);
--green-blue-crayola: hsla(203, 75%, 48%, 1);
--sap-green: hsla(86, 93%, 24%, 1); */

/* .App {
  text-align: center;
}

.App-header {
  background-color: #1d71a5;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: #e8f7eeff;
}

a {
  color: white;
}

.navbar-link {
  margin: 10px;
  color: black;
  font-size: 1.5em;
  text-decoration: none;
} */

.link {
  text-decoration: none;
  }

.navbar-link {
  text-decoration: none;
  margin-right: 1rem;
  color: #e8f7eeff;
}

.navbar-link-mobile {
  font-family: Lexend Deca;
  text-decoration: none;
  margin: .75rem;
  text-align: center;
}

.navbar-link:hover {
  font-size: larger;
}

.navbar-link-button {
  font-family: Lexend Deca;
  color: #e8f7eeff;
  text-decoration: none;
  text-align: center;
}

.material-symbols-outlined {
  padding-top: 1rem;
  padding-left: 1rem;
}