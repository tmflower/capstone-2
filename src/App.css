/** Theme colors: */

/* --honeydew: #e8f7eeff;
--ash-gray: #b8c4bbff;
--dark-brown: #6c4d25ff;
--green-blue-crayola: #1e91d6ff;
--sap-green: #447604ff; */


/* --honeydew: hsla(144, 48%, 94%, 1);
--ash-gray: hsla(135, 9%, 75%, 1);
--dark-brown: hsla(34, 49%, 28%, 1);
--green-blue-crayola: hsla(203, 75%, 48%, 1);
--sap-green: hsla(86, 93%, 24%, 1); */

/** 
-styling for selected elements by class or id to override React and/or MUI defaults
-resizing selected elements for different screen sizes
*/

.link {
  text-decoration: none;
  }

.navbar-link {
  text-decoration: none;
  margin-right: 1rem;
  color: #e8f7eeff;
}

.navbar-link-mobile {
  font-family: Lexend Deca;
  text-decoration: none;
  margin: .75rem;
  text-align: center;
  color: #1e91d6ff;
}

.navbar-link:hover {
  font-size: larger;
}

.material-symbols-outlined {
  padding-top: 1rem;
  padding-left: 1rem;
}

.emoji {
 font-size: 2.5rem;
 text-align: center;
 margin: 0;
}

.input-nonMUI {
  margin-left: 1rem;
}

.blue-text {
  color: #1e91d6ff;
}

.options-btn {
  max-width: 160px;
}

#reset-btn {
  background-color: red;

}

#navbar-simlink {
  text-transform: none;
  background-color: transparent;
  border: none;
  color: #e8f7eeff;
  font-size: 1.2rem;
  margin: 0px 0px 4px 0px;
  padding-left: 1px;
}

#navbar-simlink-mobile {
  text-transform: none;
  background-color: transparent;
  border: none;
  color: #1e91d6ff;
  font-size: 1rem;
  margin: 0px 0px 4px 0px;
}

#navbar-simlink:hover {
  font-size: larger;
}

#alphabet-button {
  font-size: 2rem;
}

#alt-button {
  background-color: #1e91d6ff;
}

#alt-button:hover {
  background-color: hsla(203, 75%, 48%, .2);
  color: #1e91d6ff;
  border: 1px solid #1e91d6ff;
}

#mock-btn {
  background-color: #1e91d6ff;
  color: #e8f7eeff;
  border-radius: 3px;
  padding: 3px 6px;
}

#giant-btn {
  background-color: hsla(203, 75%, 48%, .2);  
  border: none;
  text-transform: none;
}

#giant-btn:hover {
  background-color: hsla(203, 75%, 48%, .5);
}

@media (min-width: 375px){
    .video {
      width: 350px;
      height: 206px;
      border: 0px;
      border-radius: 10px;
    }

    .photo {
      width: 100%;
      border-radius: 10px;
    }

    .img-home {
      max-width: 8rem;
    }

    .animalIcon {
      max-width: 370px;
    }

    .Dashboard {
      max-width: 360px;
      padding: 10px;
    }

    .kid-clipart {
      width: 50px;
    }

    #modal-img {
      width:300px;
      margin: auto;
    }

    #modal-modal-title {
      font-size: 1.5rem;
      color: #1e91d6ff;
    }

    #modal-modal-description {
      font-size: 1rem;
    }
  }
  
  @media (min-width: 560px) {
    .Dashboard {
      max-width: 100%;
    }

    .table-paper {
      padding: 4%;
    }

    .kid-clipart {
      width: 80px;
    }
  }
  @media (min-width: 768px){
    .video {
      width: 500px;
      height: 294px;
    }
    .photo {
      width: 500px;
    }

    .img-home {
      max-width: 10rem;
    }
  }

  
  @media (min-width: 1024px){
    .video {
      width: 650px;
      height: 382px;
    }
    .photo {
      width: 600px;
    }
  }
  
  @media (min-width: 1200px){
    .video {
      width: 800px;
      height: 471px;
    }
  }

