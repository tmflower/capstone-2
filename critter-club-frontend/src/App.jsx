import './App.css';
import { Routes, Route } from 'react-router-dom';
import { Home } from './Home';
import { Animals } from './Animals';
import { Animal } from './Animal';
import { Quiz } from './Quiz';

export function App() {
  return (
    <div className="App">
      <header className="App-header">
        <Routes>
          <Route path="/" element={<Home />}></Route>
          <Route path="/animals" element={<Animals />}></Route>
          <Route path="/animals/:animal" element={<Animal/>}></Route>
          <Route path="/quiz" element={<Quiz />}></Route>
        </Routes>
      </header>
    </div>
  );
}