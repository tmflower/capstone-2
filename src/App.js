import './App.css';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <h2>Capstone 2</h2>
      </header>
    </div>
  );
}

export default App;
