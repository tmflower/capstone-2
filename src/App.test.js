import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { App } from './App';

// smoke test
test("renders without crashing", () => {
  render(
  <MemoryRouter>
    <App />
  </MemoryRouter>);
});

// snapshot test
test("matches snapshot", () => {
  const { asFragment } = render(
    <MemoryRouter>
      <App />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});