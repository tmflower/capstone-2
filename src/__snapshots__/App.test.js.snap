// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="MuiBox-root css-i9gxme"
    >
      <header
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-11b3ww9-MuiPaper-root-MuiAppBar-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
        />
      </header>
    </div>
    <div
      class="MuiBox-root css-ao2p2z"
    >
      <div
        class="MuiBox-root css-y1gt6f"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-1v3lxag-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
          >
            <h2
              class="MuiTypography-root MuiTypography-h2 css-1uxfxyc-MuiTypography-root"
            >
              Welcome to Critter Club!
            </h2>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-6 MuiGrid-grid-lg-4 css-o9ws9g-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 css-aqpv91-MuiPaper-root"
            >
              <img
                alt="uakari"
                class="img-home"
                src="uakari.jpeg"
              />
              <h5
                class="MuiTypography-root MuiTypography-h5 css-691kut-MuiTypography-root"
              >
                Ever heard of an uakari?
              </h5>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-6 MuiGrid-grid-lg-4 css-o9ws9g-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 css-aqpv91-MuiPaper-root"
            >
              <h5
                class="MuiTypography-root MuiTypography-h5 css-691kut-MuiTypography-root"
              >
                What does a sea dragon eat?
              </h5>
              <img
                alt="seadragon"
                class="img-home"
                src="seadragon.png"
              />
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-6 MuiGrid-grid-lg-4 css-o9ws9g-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 css-aqpv91-MuiPaper-root"
            >
              <img
                alt="lion"
                class="img-home"
                src="lion.png"
              />
              <h5
                class="MuiTypography-root MuiTypography-h5 css-691kut-MuiTypography-root"
              >
                Are all lions from Africa?
              </h5>
            </div>
          </div>
          <h4
            class="MuiTypography-root MuiTypography-h4 css-1cfo33-MuiTypography-root"
          >
            Find out the answers to these questions and SO many more when you join Critter Club!
          </h4>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-6 MuiGrid-grid-lg-4 css-o9ws9g-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 css-aqpv91-MuiPaper-root"
            >
              <h3
                class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
              >
                🧐  🧠  💭
              </h3>
              <h5
                class="MuiTypography-root MuiTypography-h5 css-vw05hv-MuiTypography-root"
              >
                Learn and test your knowledge about all kinds of animals!
              </h5>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-6 MuiGrid-grid-lg-4 css-o9ws9g-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 css-aqpv91-MuiPaper-root"
            >
              <h3
                class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
              >
                🦊  🐼  🐷
              </h3>
              <h5
                class="MuiTypography-root MuiTypography-h5 css-vw05hv-MuiTypography-root"
              >
                Earn points and collect fun animal badges!
              </h5>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-8 MuiGrid-grid-md-6 MuiGrid-grid-lg-4 css-o9ws9g-MuiGrid-root"
          >
            <div
              class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation8 css-aqpv91-MuiPaper-root"
            >
              <h3
                class="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
              >
                🔬  🏆  🥼
              </h3>
              <h5
                class="MuiTypography-root MuiTypography-h5 css-vw05hv-MuiTypography-root"
              >
                Level up from Observer to Zoologist!
              </h5>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
          >
            <a
              class="link"
              href="/signup"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Sign me up!
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
          >
            <a
              class="link"
              href="/login"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                I already signed up
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
